docker-hub/image-management.md

```diff
@@ -10,7 +10,7 @@ title: Advanced Image Management dashboard
   target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_image-mgmt"
 %}
 
-The **Advanced Image Management** dashboard allows you to manage Docker images across all repositories. [Upgrade](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"} your existing subscription to start using the Advanced Image Management dashboard.
+The **Advanced Image Management** dashboard allows you to manage Docker images across all repositories. [Upgrade](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"} your existing subscription to start using the Advanced Image Management dashboard. [Personal Access Tokens](../docker-hub/access-tokens.md) (PATs) are recommended when using automated flows and API integrations for added security. It makes it easier to manage your account if it becomes compromised.
```

Comment:
`It makes it easier to manage your account if it becomes compromised.`
I think the sentence is clear as-is, but just flagging the multiple `it` if possible to avoid.

Reply:
I'd maybe also mention why: because the account owner can easily revoke them by removing them from Hub. Otherwise LGTM

Reply:
Thank you! @craig-osterhout @milosgajdos Updated in new commit.